scripts/table_updates/config.json

```diff
@@ -19,7 +19,7 @@
         "CRC2": "syn52943210",
         "RENAL": "syn59474249"
     },
-    "main_genie_release_version": "16.6-consortium",
+    "main_genie_release_version": "17.4-consortium",
```

Comment:
@Chelsea-Na do we want to keep this at 17.2?

Reply:
Good question! If we are ready to test the output, we should test it out on 17.4-consortium.  We will eventually need to also test on 17.6-consortium once its out. 

Do we know what happens if the main GENIE value is missing? Or is the sample/patient is missing? Or if it adds to a log if there is a mismatch between the upload and the replaced value?

Reply:
This is a hard replacement so I didn't check GENIE value missingness. I checked with Rixing that we think all BPC samples/patients should be in Main GENIE referring to [BPC project description](https://sagebionetworks.jira.com/wiki/spaces/APGD/pages/2163344270/Onboarding#GENIE-BPC:~:text=Launched%20in%202019%2C%20this%20is%20an%20augmentation%20of%20the%20main%20GENIE%20project%20that%20adds%20in%20manually%20curated%20clinical%20data%20elements%20of%20a%20subset%20of%20previously%20collected%20main%20GENIE%20cases%20using%20the%20PRISSMM%20Data%20Model.). It doesn't log for the discrepancies between uploaded and replaced values. Do we want to add?

Reply:
I forgot to address your question: "Do we know what happens if a main GENIE value is missing? Or if the sample/patient is missing? If the main GENIE sample/patient is missing, the values will be filled with `NaN`. If the GENIE value is missing, then the final dataframe shows missingness since we use all main GENIE values to hard replace BPC. However, this might not be the case since

Reply:
@Chelsea-Na Two functions have been added to check for sample/patient missingness and log the discrepancies between uploaded and replaced values.